CPS-xxxx/LICENSE

```diff
@@ -0,0 +1,395 @@
+Attribution 4.0 International
```

Comment:
@rphair 

with authors adding a license like this, perhaps it does make sense to add a copyright section to CPS

Reply:
yes @Ryun1 as of https://github.com/cardano-foundation/CIPs/pull/762#issuecomment-1934233812... @stevenj please add CPS style licensing as you would for a CIP, in both the header and footer.